scripts/auto-cpufreq-run

```diff
@@ -0,0 +1,3 @@
+!#/bin/bash
```

Comment:
remove this file since we already have one like it

Reply:
It's deleted